packages/components/style.css

```diff
@@ -17,3 +17,74 @@
     @apply rounded-b-none border-b border-black/20 pb-3 dark:border-white/20;
   }
 }
+
+/* #region hamburger mobile menu */
+
+.light .nextra-sidebar-container input {
+  color: rgb(0, 52, 44);
+  background-color: rgb(248, 247, 246) !important;
+  border: 1px solid rgb(241, 238, 228);
+}
+
+.light .nextra-sidebar-container input::placeholder {
+  color: rgba(36, 88, 80, 0.9) !important;
+}
+
+.light .nextra-sidebar-container .nextra-sidebar-footer:is(html[class] *) {
+  border-color: #e5e7eb;
+}
+
+.light .nextra-sidebar-container kbd {
+  @apply !border-gray-200 !bg-white;
+}
+
+.light .nextra-sidebar-container li > :is(a, button) {
+  @apply text-gray-500 hover:bg-gray-100 hover:text-gray-900;
+}
+
+.light .nextra-sidebar-container ul::before {
+  @apply !bg-gray-200;
+}
+
+/* #endregion hamburger mobile menu */
+
+/* #region search results */
+
+/* without this, if your rotate your phone to horizontal with menu open, you see just a black overlay */
+._fixed._z-10:has(+ .hive-search-results) {
+  @apply md:hidden;
+}
+
+.light .hive-search-results {
+  @apply !border-green-200;
+  background-color: white;
+  padding: 1rem 0;
+}
+
+.light .hive-search-results .dark\:_text-gray-400:is(html[class~='dark'] *) {
+  @apply !text-green-800;
+}
+
+.light .hive-search-results .dark\:_text-gray-300:is(html[class~='dark'] *) {
+  @apply !text-green-1000;
+}
+
+.light .hive-search-results .dark\:_text-gray-300:is(html[class~='dark'] *) {
+  @apply !text-green-800;
+}
+
+.light .hive-search-results .dark\:_text-neutral-400:is(html[class~='dark'] *) {
+  color: rgb(107, 114, 128);
+}
+
+.light .hive-search-results,
+.light .hive-search-results:is(html[class~='dark'] *) {
+  background-color: #fff;
+  border-color: #e5e7eb;
+}
+
+.light .hive-search-results .dark\:before\:_bg-neutral-800:is(html[class~='dark'] *):before {
+  background-color: rgb(229, 231, 235);
+}
```

Comment:
this is too overcomplicated and will be broken in the future, import styles as layer https://nextra.site/docs/guide/advanced/customize-the-cascade-layers

Reply:
https://github.com/the-guild-org/docs/pull/1712 fixes this